### PR TITLE
[emoji] Add pip emoji package

### PR DESCRIPTION
**Pull request by tkmtnt7000**
Please add the following dependency to the rosdep database.

## Package name:

python3-emoji
pip/pip3 emoji

## Package Upstream Source:

https://github.com/carpedm20/emoji/

## Purpose of using this:

I want to release the package that depends on this library. 

Distro packaging links: https://github.com/jsk-ros-pkg/jsk_robot

## Links to Distribution Packages

- Ubuntu: https://packages.ubuntu.com/jammy/python3-emoji
- Debian: https://packages.debian.org/bookworm/python3-emoji
- Fedora: https://packages.fedoraproject.org/pkgs/python-emoji/python3-emoji/
- opensuse: https://software.opensuse.org/package/python3-emoji
- rhel: https://src.fedoraproject.org/rpms/python-emoji
- Pip: https://pypi.org/project/emoji/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1573,6 +1573,16 @@ python-elasticsearch:
   opensuse: [python-elasticsearch]
   ubuntu:
     bionic: [python-elasticsearch]
+python-emoji-pip:
```
Comment (ivanpauno) [inline rosdep/python.yaml:1576]: do you want to make it available for python3?
In that case, the key should be `python3-emoji-pip`.

Comment (ivanpauno) [reply rosdep/python.yaml:1576]: There's an ubuntu debian package available: https://packages.ubuntu.com/jammy/python3-emoji.

So consider adding a `python3-emoji` key instead

**Review by tfoote — CHANGES_REQUESTED** [review]
If it's available from debian packages we should use that and not the pip rules. 

Thanks for the contribution of rosdep keys. Please take a moment to update your PR following our rosdep rule [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions). You can add changes to the branch you have submitted from or edit the description. When you make the changes please comment back on this PR so that we get a notification that you've updated it.

**Comment by tkmtnt7000** [comment]
Thank you for quick reply.
Changed to use debian package if debian package is released.

**Review by tfoote — COMMENTED** [review]
Thanks for the quick update. I see that it's also available for debian as of bookworm. https://packages.debian.org/bookworm/python3-emoji

Please use the same wildcard technique to list older version of debian via pip and the wildcard for goign forwardard. 

And the linter has identified that there's python3-emoji on rhel and python-emoji on suse.

**Review by hidmic — APPROVED** [review]
@tkmtnt7000 would you mind updating the PR description with [a package listing](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions) for future reference?

**Comment by tkmtnt7000** [comment]
I updated the PR description.

**Review by tfoote — APPROVED** [review]
lgtm

**Pull request merged**